spec/index.html

```diff
@@ -0,0 +1,56 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Spec proposal</title>
+    <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove"></script>
+    <script class='remove'>
+      // See https://github.com/w3c/respec/wiki/ for how to configure ReSpec
+      var respecConfig = {
+        specStatus: "CG-DRAFT", // FIXME: not a CG-DRAFT, ED?
+        shortName: "lws-ucs",
+        copyrightStart:  "2024",
+        // previousPublishDate:  "N/A",
+        editors: [
+          { name: "Pierre-Antoine Champin" },
+          { name: "Hadrian Zbarcea" },
+        ],
+        group: "solid",  // FIXME: https://respec.org/w3c/groups/ does not list 'lws' yet
+        wg: "Linked Web Storage",
+        maxTocLevel: 4,
+      };
```

Comment:
The `localBiblio` config (plus the change suggested in `user-stories.md`) ensures that this reference is listed in the "References" section. (Note that all references present in https://www.specref.org/ don't need to be manually added in `localBiblio`).

The `lint` config remove spurious warnings (caused by the change suggested in `glossary.md`).

```suggestion
        localBiblio: {
          RESUMABLE_UPLOADS: {
            title: "Resumable Uploads for HTTP",
            authors: ["M. Kleidl", "G. Zhang", "L. Pardue"],
            date: "3 December 2024",
            href: "https://httpwg.org/http-extensions/draft-ietf-httpbis-resumable-upload",
            status: "Draft",
            publisher: "IETF",
          }
        },
        lint: {
          "no-unused-dfns": false,
        },
      };
```